prow/github/client.go

```diff
@@ -519,6 +521,81 @@ func (c *Client) UnassignIssue(org, repo string, number int, logins []string) er
 	return nil
 }
 
+func (c *Client) TryRequestPRReview(org, repo string, number int, logins []string) (int, error) {
+	c.log("RequestPRReview", org, repo, number, logins)
+	var pr PullRequest
+	return c.request(&request{
+		method:      http.MethodPost,
+		path:        fmt.Sprintf("%s/repos/%s/%s/pulls/%d/requested_reviewers", c.base, org, repo, number),
+		requestBody: map[string][]string{"reviewers": logins},
+		exitCodes:   []int{201},
+	}, &pr)
+}
+
+func (c *Client) RequestPRReview(org, repo string, number int, logins []string) error {
+	//If any user in the 'logins' slice is not a contributor of the repo, the entire POST will fail
```

Comment:
Also, this comment is valuable information that should probably live as a godoc comment above the method name.